Jenkinsfile

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env groovy
 
 dockerfile {
+    dockerUpstreamTag = '4.1.x-latest'
     slackChannel = '#ksql-eng'
     upstreamProjects = 'confluentinc/schema-registry'
-    dockerRepos = ['confluentinc/ksql-examples', 'confluentinc/ksql-cli', 'confluentinc/ksql-clickstream-demo']
+    dockerRepos = ['confluentinc/ksql-examples', 'confluentinc/ksql-cli', 'confluentinc/ksql-clickstream-demo', 'confluentinc/cp-ksql-cli']
```

Comment:
Is `cp-ksql-cli` the image name we want?

I am ok with this (and rather keep this name), though @ybyzek advocated to rename from `ksql-cli` to `ksql` in a somewhat related context: https://github.com/confluentinc/ksql/issues/1204.

Reply:
I think we should name the new docker image `cp-ksql-cli` for a couple of reasons:

1. The `cp-` prefix keeps it consistent with other docker images provided by Confluent.
2. The `-cli` suffix keeps it consistent with the `-server` suffix for the `cp-ksql-server` docker image.

Reply:
As I said above, I am +1 regarding `cp-ksql-cli` as the name. Just saying in case it wasn't clear.